update_all_lock_files.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
This is almost the same as what's found in the `repo_manger.py` script selection 2 but without overriding with plugin_globals.

Maybe you could add another selection to the repo_manager instead of adding this bash script file?

Some parts of the file are complicated but it should be pretty easy to add this logic to the main method and ignore the rest of the file.

Reply:
Cool! Didn't notice that python script. That's great, I'll integrate the logic into there instead 👍

Reply:
@jamshale I made the change (we have ChatGPT to thank for the speed). And I included some typo fixes :-)